ledger/src/blockstore_db.rs

```diff
@@ -1437,6 +1415,34 @@ fn get_cf_options<C: 'static + Column + ColumnName>(
     options
 }
 
+/// Constructs and returns a ColumnFamilyDescriptor based on the
+/// specified ShredStorageType.
+fn new_cf_descriptor_shreds<C: 'static + Column + ColumnName>(
+    storage_type: &ShredStorageType,
+    access_type: &AccessType,
+    oldest_slot: &OldestSlot,
+    max_cf_size: &u64,
+) -> ColumnFamilyDescriptor {
+    match storage_type {
+        ShredStorageType::RocksLevel => new_cf_descriptor::<C>(access_type, oldest_slot),
+        ShredStorageType::RocksFifo => {
+            if *max_cf_size > FIFO_WRITE_BUFFER_SIZE {
+                new_cf_descriptor_fifo::<C>(max_cf_size)
+            } else {
+                warn!(
+                    " is must be greater than {} for RocksFifo.",
+                    FIFO_WRITE_BUFFER_SIZE
+                );
```

Comment:
nit: Looks like you missed adding the column name back in when you consolidated:
```
warn!("{} cf_size must be greater than {} when using ShredStorageType::RocksFifo",
    C::NAME, FIFO_WRITE_BUFFER_SIZE);
```

Reply:
Oh yep!  Thanks for spotting that.  Let me fix it!